CHANGELOG.md

```diff
@@ -6,12 +6,18 @@
 
 - Added support for `delimiters` parameter in `functions.initcap()`.
 - Added support for `functions.hash()` to accept a variable number of input expressions.
+- Added API `Session.RuntimeConfig` for getting/setting/checking the mutability of any runtime configuration.
```

Comment:
Suggest `session.conf` instead of `Session.RuntimeConfig` because a user will use the former instead of the latter.

Reply:
Sorry I didn't see this. This is a good point.